src/content/reference/react-dom/hooks/useFormStatus.md

```diff
@@ -0,0 +1,259 @@
+---
+title: useFormStatus
+canary: true
+---
+
+<Canary>
+
+The `useFormStatus` Hook is currently only available in React's canary and experimental channels. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
+
+</Canary>
+
+<Intro>
+
+`useFormStatus` is a Hook that gives you status information of the last form submission.
+
+```js
+const { pending, data, method, action } = useFormStatus();
+```
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `useFormStatus()` {/*use-form-status*/}
+
+The `useFormStatus` Hook provides status information of the last form submission.
+
+```js [[1, 5, "status.pending"]]
+import action from './actions';
```

Comment:
Is there a reason we're importing action from ./actions vs. getting it from `useFormStatus()`?  What're the differences between the action returned from `useFormStatus()` vs. using the imported one and when to use one over the other

Reply:
The `action` prop that is given in `useFormStatus` is the action we're passing to the `<form>` tag.